pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java

```diff
@@ -383,10 +385,28 @@ void run() throws PulsarAdminException {
                 "--partitions" }, description = "Number of partitions for the topic", required = true)
         private int numPartitions;
 
+        @Parameter(names = {"--topic-metadata", "-tm"}, description = "key value pair properties(a=a,b=b,c=c)",
+                required = true)
```

Comment:
required = true? IMHO, this should be optional?

Reply:
Removed